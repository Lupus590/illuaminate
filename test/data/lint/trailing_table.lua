-- config: (lint (only (syntax:table-trailing)))

local _ = {} -- ok
local _ = { 0 } -- ok
local _ = { -- ok
  0, 1,
}

local _ = {
  0
}

local _ = { 0, }
