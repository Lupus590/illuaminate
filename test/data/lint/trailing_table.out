trailing_table.lua:[10:3-10:4]: Expected trailing "," on multiline table [syntax:table-trailing]
    │
 10 │   0
    │   ^
trailing_table.lua:[13:14-13:15]: Unexpected trailing "," on single line table [syntax:table-trailing]
    │
 13 │ local _ = { 0, }
    │              ^
No errors and 2 warnings
@@ -7,8 +7,8 @@
 }
 
 local _ = {
-  0
+  0,
 }
 
-local _ = { 0, }
+local _ = { 0 }
 
